src/index.html

```diff
@@ -3,12 +3,50 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap&subset=cyrillic" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <a href="https://www.moyo.ua/" class="menu-logo">
+        <img src="images/logo.png" alt="moyo logo" >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add empty line between siblings. Apply all over the document
```suggestion
          </li>
          
          <li class="nav__item">
```